package.json

```diff
@@ -21,7 +21,8 @@
     "node": ">=14.0.0"
   },
   "dependencies": {
-    "passport-jwt": "^4.0.1"
+    "passport-jwt": "^4.0.1",
+    "sass": "^1.57.1"
```

Comment:
I think this a mistake, `sass` is used inside the web app. Also, I think that `"passport-jwt"` is also a mistake, but it does not come from this PR...

Reply:
you probably right

Reply:
Done